engine/src/collisions/narrow_phase/plugin.cpp

```diff
@@ -99,14 +108,36 @@ std::vector<ContactPointData> computeContactPoints(const cubos::core::geom::Box*
                                             .localOn1 = localOn1,
                                             .localOn2 = localOn2,
                                             .penetration = collidingWith.penetration,
-                                            .id = 0};
+                                            .normalImpulse = 0.0F,
+                                            .frictionImpulse1 = 0.0F,
+                                            .frictionImpulse2 = 0.0F,
+                                            .fid1 = featureId1,
+                                            .fid2 = featureId2};
 
             points.push_back(contact);
         }
     }
     return points;
 }
 
+/// TODO: do a check by distance if the features are unknown?
```

Comment:
I'm not sure what this refers to, but if relevant, open an issue for this and link it here. We should get used to do that. Otherwise we just tend to forget the TODOs we have around the code